src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -64,4 +96,128 @@ public void actionPerformed(ActionEvent e) {
 
         return panel;
     }
+
+    private JPanel createSpreadsheetPanel() {
+        DefaultPanel panel = new DefaultPanel();
+        GridBagConstraints c = panel.c;
+
+        c.anchor = GridBagConstraints.LINE_END;
+        panel.add(new JLabel("Spreadsheet ID: "), c);
+
+        c.gridx++;
+        c.anchor = GridBagConstraints.LINE_START;
+        c.fill = GridBagConstraints.HORIZONTAL;
+        panel.add(new JTextField(15), c);
+
+        c.anchor = GridBagConstraints.CENTER;
+        c.gridx++;
+        panel.add(new JButton("Connect"));
+        return panel;
+    };
+
+    private JPanel createWorkspacePanel() {
+        DefaultPanel panel = new DefaultPanel();
+        panel.setBorder(new CompoundBorder(new TitledBorder("Workspace"), new EmptyBorder(8, 0, 0,0)));
+        GridBagConstraints c = panel.c;
+
+        c.anchor = GridBagConstraints.LINE_END;
+        panel.add(new JLabel("Sheet: "), c);
```

Comment:
This component order is kinda odd. It would be clearer for the code reader to put code of related components close to each other. For example:

```
c.gridy = 0;
c.gridx = 0;
panel.add(new JPanel("From column: "), c);
c.gridx = 1;
panel.add(new JSpinner(...), c);

c.gridx = 2;
panel.add(new JPanel("To column: "), c);
c.gridx = 3;
panel.add(new JSpinner(...), c);
```